xonsh/built_ins.py

```diff
@@ -714,6 +715,7 @@ def load_builtins(execer=None, config=None, login=False, ctx=None):
     builtins.__xonsh_ensure_list_of_strs__ = ensure_list_of_strs
     builtins.__xonsh_list_of_strs_or_callables__ = list_of_strs_or_callables
     builtins.__xonsh_completers__ = xonsh.completers.init.default_completers()
+    builtins.__xonsh_events__ = Events()
```

Comment:
I'm still not sure where this should happen.

If there's a specific spot we can add it to built-ins, could someone find it?

Another idea is that I could implement [`sh`](https://amoffat.github.io/sh/)-like magic where events get imported like:

```python
from xonsh.events import on_precommand
```

But my gut says that's an unnecessary amount of magic.